it/test/monitoring/AuditServiceISpec.scala

```diff
@@ -0,0 +1,88 @@
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package monitoring
+
+import org.mockito.MockitoSugar.mock
+import play.api.Application
+import play.api.inject.guice.GuiceApplicationBuilder
+import play.api.libs.json.{JsError, JsSuccess, JsValue, Json, OFormat}
+import play.api.test.FakeRequest
+import test.iUtils.BaseISpec
+import uk.gov.hmrc.domain.Nino
+import uk.gov.hmrc.http.HeaderCarrier
+import uk.gov.hmrc.identityverificationquestions.models.Payslip.NationalInsurance
+import uk.gov.hmrc.identityverificationquestions.models.{AnswerDetails, CorrelationId, QuestionDataCache, QuestionWithAnswers, Score, Selection, SimpleAnswer}
+import uk.gov.hmrc.identityverificationquestions.monitoring.auditing.AuditService
+
+import java.time.Instant
+import scala.concurrent.ExecutionContext.Implicits.global
+
+class AuditServiceISpec extends BaseISpec {
+
+
+  override lazy val fakeApplication: Application = new GuiceApplicationBuilder().configure(Map("auditing.enabled" -> "true") ++ Map("auditing.consumer.baseUri.port" -> wiremockPort)).build()
+
+
+
+  "Audit Service" should {
+    implicit val hc: HeaderCarrier = HeaderCarrier()
+    val selection = Selection(Nino("AA000002D"))
+    val fakeRequest: FakeRequest[JsValue] = FakeRequest().withBody(Json.toJson(selection)).withHeaders("User-Agent" -> "identity-verification", "X-Application-ID" -> "fakeApplicationId")
+    val auditService: AuditService = fakeApplication.injector.instanceOf[AuditService]
+    val answerDetails = AnswerDetails(NationalInsurance, SimpleAnswer(""))
+    val questionDataCache = QuestionDataCache(CorrelationId(), selection, Seq(QuestionWithAnswers(NationalInsurance, Seq("", ""))), Instant.now())
+    val score = mock[Score]
+
+
+
+    "send correct data for audit type IdentityVerificationAnswer" in {
+      auditService.sendQuestionAnsweredResult(answerDetails, questionDataCache, score, None)(hc, fakeRequest, global)
+      val multifactorAuthenticationCheckAuditRecord = recoverAuditRecords("IdentityVerificationAnswer")
+      (multifactorAuthenticationCheckAuditRecord \ "detail").validate[IdentityVerificationAnswerResult] match {
+        case JsSuccess(detail, _) =>
+          detail.callingServiceApplicationID shouldBe "fakeApplicationId"
+          detail.nino shouldBe "AA000002D"
+          detail.source shouldBe "Payslip"
+        case JsError(error) => fail(s"Unable to parse IdentityVerificationAnswerResult. Error : $error")
+      }
+    }
+
+    "send correct data for audit type CircuitBreakerUnhealthyService with Nino" in {
+      auditService.sendCircuitBreakerEvent(Selection(Nino("AA000002D")),"unavailableService")
+      val multifactorAuthenticationCheckAuditRecord = recoverAuditRecords("CircuitBreakerUnhealthyService")
+      (multifactorAuthenticationCheckAuditRecord \ "detail").validate[Selection] match {
```

Comment:
I think it would be better to create a custom class here and test all attributes similar to what you have done with IdentityVerificationAnswer.